public/scss/front.scss

```diff
@@ -0,0 +1,191 @@
+
+.packeta-widget-button {
```

Comment:
Asi došlo k nějakému omylu. Takhle je to přeci horší, než výsledné CSS. Viz https://sass-lang.com/documentation/style-rules/ Připomínka se týká všech SCSS souborů.